test/sql/json/test_json_extract.test

```diff
@@ -7,6 +7,12 @@ require json
 statement ok
 pragma enable_verification
 
+# should go to our NULL
+query T
+select json_extract('{"foo": null}', '$.foo')
```

Comment:
Can we add a test where we do this with null values in a list, and with only a `null` constant? What about a JSON path expression where a value early on in the path is NULL?

Reply:
Done, works as expected